query/rds/rds_db_instance_events_subscription.sql

```diff
@@ -0,0 +1,18 @@
+select
+  -- Required Columns
+  arn as resource,
+  case
+    when source_type <> 'db-instance' then 'skip'
+    when source_type = 'db-instance' and enabled and event_categories_list @> '["failure", "maintenance", "configuration change"]' then 'ok'
+    else 'alarm'
+  end as status,
+  case
+    when source_type <> 'db-instance' then cust_subscription_id || ' is skipped for critical database instance events.'
```

Comment:
We should change the reason messaging here - let's not use the word skipped as already the status is skip

`then cust_subscription_id || ' is skipped for critical database instance events.' `can be >>> `then cust_subscription_id || ' event subscription of ' ||  source_type || '.'